client/src/components/LanguageSelector/LanguageSelector.tsx

```diff
@@ -3,18 +3,20 @@ import { UIcontext } from '../contexts/UIcontext/UIcontext';
 import './LanguageSelector.scss';
 import PropTypes from 'prop-types';
 
-const LanguageSelector = (props) => {
-    const defaultLang = localStorage.getItem('lang');
-    const [sliderPos, setSliderPos] = useState(defaultLang === 'hu' ? 'left' : 'right');
+interface LanguageSelectorProps {
+    size?: 'small'
```

Comment:
Does it mean that "size" has a default value "small"?
I dont know too much about TS interfaces, but
--> I would write here that type of 'size' is an enum, and in line 10, that default value of size is small.

Reply:
Enum would also work for sure.
I decided to use string union in many places because sometimes we display/change (like toUppercase)  the value on the interface. But not this one for sure.